### DISCUSSION TITLE
Yank and Cut problem if the files are located in different places

### DISCUSSION THREAD

**Discussion by og900aero** [sxyazi/yazi#445] [1. Q&A] [answered]
What system are you running Yazi on?
Debian 12

What terminal are you running Yazi in?
Alacritty 0.11.0

Yazi version
yazi 0.1.5 (https://github.com/sxyazi/yazi/commit/1d1a51271036a4ea2e9ec67d1c7fe5cabeb5d75a 2023-12-07)

Describe the bug

I often copy and move several files at the same time, which are in different places, and I want to paste them into one place.
I mark these files (with space), then I press y to yank, then it will mark for yanking only the file in which I am currently in the directory. Other selections that are in other directories will remain selected, but will not be marked for yanking.

To Reproduce
Example:

Mark with space the following files:

/home/Data/Pictures/1.jpg
/home/Data/Movies/2.mp4
/home/Data/Music/3.flac
/home/Data/Car/4.txt
Then press y for yank, and mark for yank ony the 4.txt.

I dont know this is a bug, or "feature", because yank only the current directories selected files.

**Comment by sxyazi** [comment, marked as answer]
I think your requirement is covered in this RFC: https://github.com/sxyazi/yazi/issues/319.

We are exploring how to implement it, feel free to leave your thoughts on that RFC.

**Reply by og900aero** [reply]
Yes, that will be it. I will follow this request.
